.circleci/config.yml

```diff
@@ -201,6 +201,26 @@ jobs:
             export SIMULATE_WITHOUT_LEDGER=1
             just simulate
 
+  just_simulate_006-ecotone-set-gas-config:
```

Comment:
What do you think of calling these tasks `005-01-protocol-versions-ecotone` and `005-02-ecotone-set-gas-config`? The rationale being that task `006-...` will keep it's numbering corresponding to [upgrade 6](https://gov.optimism.io/t/upgrade-proposal-6-multi-chain-prep-mcp-l1/7677)

Reply:
renamed this one to `005-2`, can I leave the other as is?